vscode-dotnet-runtime-library/src/Acquisition/DotnetConditionValidator.ts

```diff
@@ -152,40 +167,55 @@ Please set the PATH to a dotnet host that matches the architecture ${requirement
                 const availablePatchStr: string | null = requirement.acquireContext.mode !== 'sdk' ? versionUtils.getRuntimePatchVersionString(availableVersion, this.workerContext.eventStream, this.workerContext)
                     : versionUtils.getSDKCompleteBandAndPatchVersionString(availableVersion, this.workerContext.eventStream, this.workerContext);
                 const availablePatch = availablePatchStr ? Number(availablePatchStr) : null;
-                switch (requirement.versionSpecRequirement)
+
+                const availableBandStr = requirement.acquireContext.mode === 'sdk' ? versionUtils.getFeatureBandFromVersion(availableVersion, this.workerContext.eventStream, this.workerContext, false) ?? null : null;

```

Comment:
So, is `?? null` saying, "If it is null or undefined, return null"? If `getFeatureBandFromVersion` cannot return `undefined`, it doesn't do anything.

Reply:
This was supposed to handle a return value of '' which evaluates to false in JS. But even though '' is false, that doesn't mean ?? will convert it to null -- good catch.